src/ducks/apps/components/Sections/components/DropdownFilter.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+
+import SelectBox from 'cozy-ui/react/SelectBox'
+import Icon from 'cozy-ui/react/Icon'
+import PropTypes from 'prop-types'
+
+const SmallArrow = () => (
+  <Icon
+    className="sto-sections-dropdown-icon"
+    icon="small-arrow"
+    color="#95999D"
```

Comment:
I know this is not part of this PR but : coolGrey ;)

Reply:
Done in https://github.com/cozy/cozy-store/pull/622/commits/b1367b25992e4d1363f01db9adbc6f91bd33c625 (`var(--coolGrey)`).